frontend/src/libs/components/link/link.tsx

```diff
@@ -3,13 +3,24 @@ import { NavLink } from 'react-router-dom';
 import { type AppRoute } from '#libs/enums/enums.js';
 import { type ValueOf } from '#libs/types/types.js';
 
+import styles from './link.module.scss';
+
 type Properties = {
   to: ValueOf<typeof AppRoute>;
   children: React.ReactNode;
+  className?: string;
 };
 
-const Link: React.FC<Properties> = ({ children, to }: Properties) => (
-  <NavLink to={to}>{children}</NavLink>
+const stylePatcher = (newStyle: string): string => `${styles.link} ${newStyle}`;
```

Comment:
This function can be used in other components as well. Move it to helpers folder